src/cbc_sdk/endpoint_standard/recommendation.py

```diff
@@ -0,0 +1,473 @@
+#!/usr/bin/env python3
+
+# *******************************************************
+# Copyright (c) VMware, Inc. 2020-2021. All Rights Reserved.
+# SPDX-License-Identifier: MIT
+# *******************************************************
+# *
+# * DISCLAIMER. THIS PROGRAM IS PROVIDED TO YOU "AS IS" WITHOUT
+# * WARRANTIES OR CONDITIONS OF ANY KIND, WHETHER ORAL OR WRITTEN,
+# * EXPRESS OR IMPLIED. THE AUTHOR SPECIFICALLY DISCLAIMS ANY IMPLIED
+# * WARRANTIES OR CONDITIONS OF MERCHANTABILITY, SATISFACTORY QUALITY,
+# * NON-INFRINGEMENT AND FITNESS FOR A PARTICULAR PURPOSE.
+
+"""Model and query APIs for Recommendations"""
+
+from cbc_sdk.base import (UnrefreshableModel, BaseQuery, CriteriaBuilderSupportMixin, IterableQueryMixin,
+                          AsyncQueryMixin)
+from cbc_sdk.errors import ApiError, NonQueryableModel
+from cbc_sdk.platform.devices import DeviceSearchQuery
+import logging
+
+
+log = logging.getLogger(__name__)
+
+
+"""Recommendation models"""
+
+
+class Recommendation(UnrefreshableModel):
+    """Represents a recommended proposed policy change for the organization."""
+    urlobject = "/recommendation-service/v1/orgs/{0}/recommendation"
+    urlobject_single = "/recommendation-service/v1/orgs/{0}/recommendation/{1}"
+    primary_key = "recommendation_id"
+    swagger_meta_file = "endpoint_standard/models/recommendation.yaml"
+
+    def __init__(self, cb, model_unique_id, initial_data=None):
+        """
+        Initialize the Recommendation object.
+
+        Args:
+            cb (BaseAPI): Reference to API object used to communicate with the server.
+            model_unique_id (str): ID of the recommendation represented.
+            initial_data (dict): Initial data used to populate the recommendation.
+        """
+        super(Recommendation, self).__init__(cb, model_unique_id, initial_data, full_doc=True)
```

Comment:
If I saw correctly there is no /recommendation-service/v1/orgs/{org_key}/recommendation/{id}, but just search, so maybe we do not need model_unique_id. If you do not think that's correct just disregard.

Reply:
It gets initialized when those objects get created in the query, with the `recommendation_id` member of the item data. And I _do_ need it when putting together the URL in the `_take_action` function.